docs/usage.md

```diff
@@ -8,100 +8,83 @@
 
 The pipeline is divided into two parts:
 
-1. Downloading and building the references, using `--build_references`: done only once and after each update.
-2. Detecting fusions using (any combination of) the following tools:
-   - arriba
-   - fusioncatcher
-   - pizzly
-   - squid
-   - starfusion
-3. QC and visualisation tools
-   - Fastqc
-   - MultiQC
-   - arriba visualisation (for fusion detected by arriba only)
-   - fusion-report
-   - fusionInspector
-
-### Prerequisite: download and build references
+1. Download and build references 
+   - specified with `--build_references` parameter
+   - this is required only once before running the pipeline. 
+   - It should be rerun after pipeline updates.
+2. Detecting fusions 
+   - This uses any combination of the fusion tools: arriba, fusioncatcher, pizzly, squid, starfusion.
+   - QC is performed with: Fastqc and MultiQC.
+   - Visualisation of fusions is performed with: arriba visualisation (for fusion detected by arriba only), fusion-report, fusionInspector
+
+### 1. Download and build references
 
 The rnafusion pipeline needs references for the fusion detection tools, so downloading these is a **requirement**.
-It is possible to download and build each reference manually (for example in case of non-human samples that are not supported currently) feed references manually to rnafusion using the arguments `--<tool>_ref` but it is advised to download references with rnafusion:
+Whilst it is possible to download and build each reference manually, it is advised to download references with the rnafusion pipeline.
+
+First register for a free account at COSMIC at https://cancer.sanger.ac.uk/cosmic/register using your university email ending .ac.uk. The account is only activated upon clicking the link in the registation email.
+
+Then run the pipeline, ensuring to include in the command line your COSMIC registration email and password as well as the path to the desired reference output directory:
 
 ```bash
 nextflow run nf-core/rnafusion \
 --build_references --all \
---genomes_base <PATH> \
+--cosmic_username <EMAIL> --cosmic_passwd <PASSWORD> \
+--outdir <PATH> -r 2.0.0
```

Comment:
```suggestion
nextflow run nf-core/rnafusion \
  --build_references --all \
  --cosmic_username <EMAIL> --cosmic_passwd <PASSWORD> \
  --outdir <PATH>
```

Reply:
`--genomes_base` is needed to know where to write the references